tests/baselines/reference/privateNamesAndIndexedAccess.errors.txt

```diff
@@ -0,0 +1,40 @@
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(7,28): error TS1005: ']' expected.
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(7,28): error TS2300: Duplicate identifier '#bar'.
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(7,28): error TS2717: Subsequent property declarations must have the same type.  Property '#bar' must be of type 'number', but here has type 'any'.
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(7,32): error TS1005: ';' expected.
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(7,34): error TS1068: Unexpected token. A constructor, method, accessor, or property was expected.
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(7,36): error TS7008: Member '3' implicitly has an 'any' type.
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(9,28): error TS2339: Property '#bar' does not exist on type 'C'.
+tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts(11,1): error TS1128: Declaration or statement expected.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNamesAndIndexedAccess.ts (8 errors) ====
+    class C {
+        foo = 3;
+        #bar = 3;
+        constructor () {
+            const ok: C["foo"] = 3;
+            // not supported yet, could support in future:

```

Comment:
`C[#foo]` isn't allowed in JS.

We *may* want syntax like this at the type level in future: `const x: C[#foo]` (@DanielRosenwasser pointed this out).
But we punted on a type-level syntax, because we would want to use something familiar that isn't clashy and work on runtime metaprogramming for private names (decorators proposal I think) is still in flux.